### PR TITLE
Add initial schedules

### PR DESCRIPTION
**Pull request by jon-bell**

### CODE REVIEW

File: _schedules/s2.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    Thu:
+      date: 2022/11/24
+      content: thanksgiving
+  '13':
+    Mon:
+      date: 2022/11/28
+      content: 12b
+    Thu:
+      date: 2022/12/01
+      content: 13a
```
Comment (iamabhutta) [inline _schedules/s2.md]: 12/1 should be Presentations, not work on project (For Section 2). Rest of the calendar seems fine. Thanks

**Review by iamabhutta — COMMENTED** [review]
updated file to make 12/1 as presentation day. Hope my change didn't mess up anything else

**Comment by mwand** [comment]
lgtm.

**Comment by jon-bell** [comment]
Thank you for the review!

**Pull request merged**